pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -229,6 +229,7 @@ var _ = Describe("v1beta1 inference service controller", func() {
 								"serving.kserve.io/autoscalerClass":                        "hpa",
 								"serving.kserve.io/metrics":                                "cpu",
 								"serving.kserve.io/targetUtilizationPercentage":            "75",
+								"service.beta.openshift.io/serving-cert-secret-name":       "raw-foo-predictor-serving-cert",
```

Comment:
to align to the other tests, can we use this?
```suggestion
								"service.beta.openshift.io/serving-cert-secret-name":     predictorDeploymentKey.Name + constants.ServingCertSecretSuffix,
```

Reply:
done